source/_snippet.html.erb

```diff
@@ -0,0 +1,4 @@
+<div class="snippet">
+  <h3 class="snippet-title"><%= language %></h3>
```

Comment:
I’d suggest that these `h3` headings be removed, or somehow made more unique.

Right now, the HTML outline produced will include three `h3` elements _per component_. So once we have a page with more than one component, these headings become less descriptive and it is hard to parse which headings belong to which example.

Couple thoughts:
- Are individual headings needed?
- Could the `h3` be something like “Code for Example Component”?
- Could we utilize visually hidden text or `aria-label` to override how this is perceived in things like screen readers, so that it would be “HTML _for Example Component_” and not just “HTML”.